### DISCUSSION TITLE
clean up after kernel switch?

### DISCUSSION THREAD

**Discussion by bobblkabb** [zbm-dev/zfsbootmenu#317] [Questions] [answered]
I have now changed from kernel 5.15 to 5.18, but after each update the dkms modules for 5.15 are reprocessed.
How can I clean up the system?

**Comment by ahesford** [comment, marked as answer]
By removing the 5.15 headers or the DKMS module for that series, but these details are a question for your distribution and have nothing to do with ZFSBootMenu.
